### PR TITLE
Build skeleton of Enrolled Course

### PR DESCRIPTION
**Pull request by LloydFrancisco**

### CODE REVIEW

File: src/app/(views)/enrolled-courses/[courseId]/modules/page.tsx [hunk 1]
```diff
@@ -5,40 +5,142 @@ import Link from 'next/link'
...
+export default function Course() {
   const params = useParams()
   const { courseId } = params as { courseId: string }
-
   const { courses, fetchingCourses } = useGetCourses({ courseId })
+  const course = courses[0]
 
-  if (fetchingCourses) {
-    return <Loading />
+  if (fetchingCourses || !course) {
```
Comment (jaymercado) [inline src/app/(views)/enrolled-courses/[courseId]/modules/page.tsx:47]: If there's no course `!course `, why are displaying a loading component?
If we finished loading and there's no course, we should display an error message.

File: src/app/(views)/enrolled-courses/[courseId]/modules/page.tsx [hunk 2]
```diff
@@ -5,40 +5,142 @@ import Link from 'next/link'
...
+                    style={{ textDecoration: 'none' }}
+                  >
+                      <span className="fw-semibold">{module.title}</span>
+                      <small
+                        className="d-block text-truncate text-secondary"
+                        style={{ maxWidth: '150px' }}
+                      >
+                        {module.description}
+                      </small>
+                    </Link>{' '}
```
Comment (jaymercado) [inline src/app/(views)/enrolled-courses/[courseId]/modules/page.tsx]: Do we need this`{' '}`

File: src/app/(views)/enrolled-courses/[courseId]/page.tsx [hunk 3]
```diff
@@ -14,48 +45,102 @@ export default function Course() {
...
+                    style={{ textDecoration: 'none' }}
+                  >
+                      <span className="fw-semibold">{module.title}</span>
+                      <small
+                        className="d-block text-truncate text-secondary"
+                        style={{ maxWidth: '150px' }}
+                      >
+                        {module.description}
+                      </small>
+                    </Link>{' '}
```
Comment (jaymercado) [inline src/app/(views)/enrolled-courses/[courseId]/page.tsx:119]: Do we need this `{' '}`

File: src/app/(views)/enrolled-courses/page.tsx [hunk 4]
```diff
@@ -21,7 +21,9 @@ export default function ManagedCourses() {
 
       <div className="d-grid gap-4" style={{ gridTemplateColumns: 'repeat(auto-fill, minmax(200px, 1fr))', justifyContent: 'left' }}>
         {fetchingCourses ? (
+          <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '60vh' }}>
```
Comment (jaymercado) [inline src/app/(views)/enrolled-courses/page.tsx]: If you're going to add ` <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '60vh' }}>` everytime, you might as well put it inside `<Loading />`,  no?

**Review by jaymercado — CHANGES_REQUESTED** [review]

File: src/app/(views)/enrolled-courses/page.tsx (hunk 4 above)
Comment (LloydFrancisco) [reply src/app/(views)/enrolled-courses/page.tsx]: true, but I have yet to explore find where that it

Comment (LloydFrancisco) [reply src/app/(views)/enrolled-courses/page.tsx]: kita ko na

File: src/components/Loading.tsx [hunk 5]
```diff
@@ -2,7 +2,10 @@ import React from 'react'
 
 export default function Loading() {
   return (
-    <div className="flex justify-center items-center h-screen">
+    <div 
+      className="flex justify-center items-center h-screen" 
+      style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '60vh' }}
```
Comment (jaymercado) [inline src/components/Loading.tsx]: no inline styling

**Review by jaymercado — CHANGES_REQUESTED** [review]

**Review by jaymercado — CHANGES_REQUESTED** [review]
Remove all inline styling.

**Pull request closed without merging**